syntax = "proto3";
package Jde.Proto;

enum ELogLevel{
	Trace = 0;
	Debug = 1;
	Information = 2;
	Warning = 3;
	Error = 4;
	Critical = 5;
	LogLevelNone = 6;
	NoLog = -1;
}